mantidproject/mantid
# Add container subtraction tab to indirect

Fixes #13110 

The new ContainerSubtraction tab is now in the Indirect Corrections tab.
# To Test
- Open the ContainerSubtraction tab (Interfaces > Indirect > Corrections > ContainerSubtraction)
- Select a suitable `Sample` File and matching `Container` file.
- Optionally Scale the container
- Run the Subtraction
- Several features need to be checked to ensure that the new tab functions correctly:
  - The plot options plot the correct things
  - The plot preview displays correct and reasonable data
    - This can be checked against the ApplyPaalmanPings tab (without using corrections only can). The can subtraction will be removed from the ApplyPaalmanPings tab at a later date.
  - The relevant files appear in the ADS 
    - Original files
    - Wavelength converted versions of the originals (if they were not in wavelength originally)
    - The container subtraction

This has should now feature in the Indirect_Corrections.rst

ToDo:
- [x] update release notes (http://www.mantidproject.org/Release_Notes_3_5_Indirect_Inelastic#New_features)
